Kitodo-API/src/main/java/org/kitodo/config/OPACConfig.java

```diff
@@ -518,6 +516,15 @@ public static XMLConfiguration getKitodoOpacConfiguration() throws Configuration
             String message = "File not found: " + kitodoConfigOpacFile.getAbsolutePath();
             throw new ConfigException(message, new FileNotFoundException(message));
         }
-        return new XMLConfiguration(kitodoConfigOpacFile.getFile());
+        // Create and initialize the builder
+        ReloadingFileBasedConfigurationBuilder<XMLConfiguration> builder =
+            new ReloadingFileBasedConfigurationBuilder<>(XMLConfiguration.class)
+                .configure(new Parameters().xml()
+                    .setFile(kitodoConfigOpacFile.getFile())
+                    .setListDelimiterHandler(new DefaultListDelimiterHandler('&')));
+        // Register an event listener for triggering reloading checks
+        builder.addEventListener(ConfigurationBuilderEvent.CONFIGURATION_REQUEST,
+            (EventListener) event -> builder.getReloadingController().checkForReloading(null));
```

Comment:
```suggestion
            event -> builder.getReloadingController().checkForReloading(null));
```
Is this cast necessary? See above.